README.md

```diff
@@ -121,6 +121,21 @@ animator.start();
 animationView.cancelAnimation();
 ```
 
+You can add a color filter to the whole animation, a specific layer, or specific content within a layer:
```

Comment:
Can you add a sentence that indicates that the content includes things like fill, stroke, or group, not a shape?